src/snap-confine.apparmor.in

```diff
@@ -21,6 +21,8 @@
     /usr/lib/@{multiarch}/libseccomp.so* mr,
     /lib/@{multiarch}/libseccomp.so* mr,
 
+    @{PROC}/@{pid}/auxv r,
+
```

Comment:
The updated rules should absolutely be in upstream and thank you for proposing them! My only question is about the particular auxv rule. I also don't see it in 1.0.38-0ubuntu0.16.04.10 from xenial-proposed. Can you compare this PR with the one that you said was uploaded? At this point I checked the image ppa, xenial-proposed and an image mvo gave me that has the unsafe rules but not auxv.